### PR TITLE
User post models

### PR DESCRIPTION
**Pull request by wathigo**
Tasks Description
1. Creation of user and post models. 
2. Implementation of integration tests.
3. Implementation of views.
Please review this PR. Thank you.

### CODE REVIEW

File: facebook/Gemfile [hunk 1]
```diff
@@ -49,6 +55,8 @@ group :development do
   # Access an interactive console on exception pages or by calling 'console' anywhere in the code.
   gem 'web-console', '>= 3.3.0'
   gem 'rspec-rails', '~> 3.8'
+  gem 'factory_bot_rails'
+  gem 'capybara', '>= 2.2'
```
Comment (efrapp) [inline facebook/Gemfile]: You have this gem duplicated in line 68

Comment (wathigo) [reply facebook/Gemfile]: Thank you @efrapp for reviewing my PR. I will add the user index page and also fix the duplicates in the gemfile.

**Comment by wathigo** [comment]
Hi @efrapp. I have implemented the required changes. Please review the changes.

File: fb/app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+class User < ApplicationRecord
+  has_many  :created_posts, class_name: 'Post',
+                             foreign_key: 'creator_id',
+                             dependent: :destroy
+  scope :all_except, ->(user) { where.not(id: user) }
```
Comment (mcrd25) [inline fb/app/models/user.rb]: smart

Comment (wathigo) [reply fb/app/models/user.rb]: Thank you

**Comment by wathigo** [comment]
Thank you @mcrd25 for reviewing my PR. 
Yes, am working alone on this project. We agreed to work alone with my partner after which we sought an approval from Willow.
I have read all you suggestions and I think they are all important. I will implement the suggestions after which I will merge the branch to develop.

**Pull request merged**